src/Dropzone/templates/form_theme.html.twig

```diff
@@ -2,23 +2,15 @@
     {%- set dataController = (attr['data-controller']|default('') ~ ' symfony--ux-dropzone--dropzone')|trim -%}
     {%- set attr = attr|merge({'data-controller': '', class: (attr.class|default('') ~ ' dropzone-input')|trim}) -%}
 
-    <div class="dropzone-container" data-controller="{{ dataController }}">
+    <div class="dropzone-container" data-controller="{{ dataController }}" data-symfony--ux-dropzone--dropzone-target="container">
         <input type="file" {{ block('widget_attributes') }} data-symfony--ux-dropzone--dropzone-target="input" />
-
         <div class="dropzone-placeholder" data-symfony--ux-dropzone--dropzone-target="placeholder">
             {%- if attr.placeholder is defined and attr.placeholder is not none -%}
                 {{- translation_domain is same as(false) ? attr.placeholder : attr.placeholder|trans({}, translation_domain) -}}
             {%- endif -%}
         </div>
-
+        <button class="dropzone-preview-button" type="button" data-symfony--ux-dropzone--dropzone-target="previewClearButton"></button>
```

Comment:
This is problematic, as people would have a new button that appears on their forms / UI..

I think  we do need to create a new one